places.geojson

```diff
@@ -46,6 +46,15 @@
                 "location": "Morro Bay, California, USA",
                 "description": "An ancient volcanic plug formed by magma, this 581-foot rock is known as a harbor entrance landmark (also a place which Chris visited often when he was young)."
             }
-        }        
+        },
+        {
+          "type": "Feature",
+          "geometry": { "type": "Point", "coordinates": [ -42.0391845703125,-22.98873816096073] },
+          "properties": {
+            "name": "Cabo Frio Location",
+            "type: "Upwelling zone",

```

Comment:
Thanks @nilodna ! I think this needs a quotation mark
```suggestion
            "type": "Upwelling zone",

```

Reply:
Thanks @abarciauskas-bgse. I think I solved it now.